commands/deploy.js

```diff
@@ -359,17 +360,15 @@ function logDetails(details, opts) {
       .value();
   }
 
-  if(details.runTestResult) {
+  if(details.runTestResult.any) {
```

Comment:
In prior versions of the API, I'm fairly certain that `details.runTestResult` was where the test data was returned from versus `details.runTestResult.any`.  In case this is the case, we'd need some sort of conditional to handle this backwards compatibility. For instance...

```js
var runTestResult = (details.runTestResult && details.runTestResult.any) ? details.runTestResult.any : details.runTestResult;

if(runTestResult) { 
...
```